src/lib.rs

```diff
@@ -732,9 +740,11 @@ mod tests {
         UserCreated,
 
         #[strum(serialize = "invalid.schema")]
+        #[cfg(feature = "mock")]
         InvalidSchema,
 
         #[strum(serialize = "invalid.route")]
+        #[cfg(feature = "mock")]
         InvalidRoute,
```

Comment:
This could happen regardless of mock publisher I think.

Reply:
When a test is written to use this message with `!= mock` you can remove/adapt this cfg. For now all this variant does without the mock feature is making the compiler to raise a fair amount of dead code warnings.

Reply:
ah this is in test code, never mind then.